src/hal/user_comps/mb2hal/mb2hal_HOWTO.ini

```diff
@@ -94,6 +94,7 @@ SERIAL_DELAY_MS=10
 MB_SLAVE_ID=1
 
 #REQUIRED: The first element address.
+#The address is a decimal number
```

Comment:
Numbers including a comma is not allowed here afaik
```suggestion
#The address is an integer number
```

Reply:
"Integer" will not help mb2hal begginer user.

This is typical page from driver manual:
![Delta_C2000-manual_modbus](https://user-images.githubusercontent.com/96618597/222678400-66601831-82aa-4b70-98d4-0967282b69bc.png)
Integer can be sometimes similar hexadecimal number.
I want make help, that user has to transform 2000H => 8192D, and then he must write only "8192"
This will help too:
https://www.binaryhexconverter.com/hex-to-decimal-converter
I am spend many time for find out it.


A fantastic thing would be if mb2hal could process xxxxH and convert it to a decimal number by itself. But this is beyond my programming capabilities.

Reply:
My first encounter with Modbus was in LCNC using the mb2hal component. It took me a lot of time before I broke the first communication. There was a lot I had to study. One of the things that I lost a lot of time on was that the address should be converted from hexadecimal to decimal. Once I know it, I'll just convert it.

Can we write:
#The address is decimal system
or
#The address must be converted from hexagonal to decimal system
#you can use: https://www.binaryhexconverter.com/hex-to-decimal-converter
?

What do you suggest?

Reply:
`#REQUIRED: The first element address (decimal integer)`

Reply:
#REQUIRED: The first element address (decimal integer)
I agree with it. Could you change it? Git and Github is not my strong point.